src/components/CaseDetails/styles.tsx

```diff
@@ -5,7 +5,7 @@ import COLORS from '@/styles/colors';
 export const CaseDisplay = styled.aside`
   position: relative;
   width: 100%;
-  margin-bottom: 0.5rem;
+  /* margin-bottom: 0.5rem; */ //idk if I need this
```

Comment:
you probably dont...

Reply:
I thought I had gotten rid of all the comments lol, nice catch